hazelcast/src/main/java/com/hazelcast/jet/impl/MasterJobContext.java

```diff
@@ -407,6 +417,14 @@ Tuple2<CompletableFuture<Void>, String> requestTermination(
         return result;
     }
 
+    @Nonnull
+    Tuple2<CompletableFuture<Void>, String> requestTermination(
```

Comment:
@viliam-durina  this method will not be needed after EE PR is merged. Can I remove now, or only after merging EE changes?

Reply:
yes. I left it so EE compiles after OS merge without EE branch merge. but if it is possible to do these merges at the same time, then the method is can be removed but it will probably cause `pr-EE-compiler` job failure

Reply:
removed